frontend/src/static/css/config/_dark_theme.scss

```
@@ -187,24 +187,24 @@ body.dark_theme {
+	--playlist-view-actions-bg-color: var(--user-action-form-inner-bg-color);
 
 	// --playlist-view-media-bg-color: #292929;
-	--playlist-view-media-bg-color: var(--sidebar-bg-color);
+	--playlist-view-media-bg-color: var(--body-bg-color);
 
 	--playlist-view-media-order-number-color: rgb(136, 136, 136);
 
 	--playlist-view-item-title-text-color: #fff;
 
 	--playlist-view-item-author-text-color: #fff;
 	// --playlist-view-item-author-bg-color: #292929;
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)